packages/usa-tooltip/src/index.js

```diff
@@ -353,37 +372,25 @@ const setUpAttributes = (tooltipTrigger) => {
   return { tooltipBody, position, tooltipContent, wrapper };
 };
 
+/**
+ * Set up tooltip trigger event handlers.
+ * @param {HTMLElement} tooltipTrigger The element that creates the tooltip
+ */
+const setUpEvents = (tooltipTrigger) => {
+  tooltipTrigger.addEventListener("mouseover", onTooltipActivate);
+  tooltipTrigger.addEventListener("focusin", onTooltipActivate);
+  tooltipTrigger.addEventListener("mouseout", onTooltipDeactivate);
+  tooltipTrigger.addEventListener("focusout", onTooltipDeactivate);
+};
+
 // Setup our function to run on various events
 const tooltip = behavior(
-  {
-    "mouseover focusin": {
-      [TOOLTIP](e) {
-        const trigger = e.target;
-        const elementType = trigger.nodeName;
-
-        // Initialize tooltip if it hasn't already
-        if (elementType === "BUTTON" && trigger.hasAttribute("title")) {
-          setUpAttributes(trigger);
-        }
-      },
-      [TOOLTIP_TRIGGER](e) {
-        const { trigger, body } = getTooltipElements(e.target);
-
-        showToolTip(body, trigger, trigger.dataset.position);
-      },
-    },
-    "mouseout focusout": {
-      [TOOLTIP_TRIGGER](e) {
-        const { body } = getTooltipElements(e.target);
-
-        hideToolTip(body);
-      },
-    },
-  },
+  {},
   {
     init(root) {
       selectOrMatches(TOOLTIP, root).forEach((tooltipTrigger) => {
         setUpAttributes(tooltipTrigger);
+        setUpEvents(tooltipTrigger);
       });
     },
     setup: setUpAttributes,
```

Comment:
Since we're exporting `setUpAttributes`, I think it would make sense to export `setUpEvents` as well.

Reply:
> Since we're exporting `setUpAttributes`, I think it would make sense to export `setUpEvents` as well.

Are you thinking `setup` would call both `setUpAttributes` and `setUpEvents`, or a separate property for just `setUpEvents` ? It's not clear to me how the `setup` public API is meant to be used.

I'm thinking maybe we could do something like this? What do you think?

```ts
function setup(tooltipTrigger) {
  setUpAttributes(tooltipTrigger);
  setUpEvents(tooltipTrigger);
}

// ...

const tooltip = behavior(
  {
    init(root) {
      selectOrMatches(TOOLTIP, root).forEach(setup);
    },
    setup,
    // ...
  },
);
```

Reply:
I couldn't recall the original motivation for creating a separate "setUpEvents" function aside from the fact that these event handlers aren't "attributes" as such. But based on your comment, seeing how `setup` should fully set up the component, I decided to just move the event binding into the existing function and rename it from `setUpAttributes` to just `setup`. I think that should simplify things overall and ensure that "setup" does all of the setup pieces.